diff --git a/ui/frob-prompt.c b/ui/frob-prompt.c
index 20462f6..11b1346 100644
--- a/ui/frob-prompt.c
+++ b/ui/frob-prompt.c
@@ -25,7 +25,7 @@
 #include "gcr/gcr-base.h"

 #include <gtk/gtk.h>
-#include <gdk/gdkx.h>
+#include <gdk/gdk.h>

 #include <err.h>

diff --git a/ui/frob-system-prompt.c b/ui/frob-system-prompt.c
index e08114f..aead257 100644
--- a/ui/frob-system-prompt.c
+++ b/ui/frob-system-prompt.c
@@ -26,7 +26,7 @@
 #include "egg/egg-testing.h"

 #include <gtk/gtk.h>
-#include <gdk/gdkx.h>
+#include <gdk/gdk.h>

 #include <unistd.h>
 #include <string.h>
diff --git a/ui/gcr-prompt-dialog.c b/ui/gcr-prompt-dialog.c
index 48088d7..0931ae2 100644
--- a/ui/gcr-prompt-dialog.c
+++ b/ui/gcr-prompt-dialog.c
@@ -27,7 +27,7 @@
 #include "gcr-secure-entry-buffer.h"

 #include <gtk/gtk.h>
-#include <gdk/gdkx.h>
+#include <gdk/gdk.h>
 #include <glib/gi18n.h>

 /**
diff --git a/ui/gcr-prompter-tool.c b/ui/gcr-prompter-tool.c
index 0a22238..0f3ec59 100644
--- a/ui/gcr-prompter-tool.c
+++ b/ui/gcr-prompter-tool.c
@@ -27,7 +27,7 @@

 #include <glib/gi18n.h>
 #include <gtk/gtk.h>
-#include <gdk/gdkx.h>
+#include <gdk/gdk.h>
 #include <pango/pango.h>

 #include <locale.h>
